nestkernel/completed_checker.h

```diff
@@ -95,9 +95,9 @@ class CompletedChecker
   void set( const thread tid, const bool v );
 
   /**
-   * Returns const reference toelement at position tid.
+   * Returns const reference to element at position tid.
    */
-  const bool& operator[]( const thread tid ) const;
+  bool operator[]( const thread tid ) const;
```

Comment:
why did you need to remove the const here?

Reply:
@jakobj Instead of returning a const reference to a bool here, we return simply a bool as there is no reason to return a bool as a reference. If the resulting bool should be constant, that should be handled by the caller.